llvm/include/llvm/IR/Attributes.h

```diff
@@ -1292,7 +1292,8 @@ bool isNoFPClassCompatibleType(Type *Ty);
 /// if only attributes that are known to be safely droppable are contained in
 /// the mask; only attributes that might be unsafe to drop (e.g., ABI-related
 /// attributes) are in the mask; or both.
```

Comment:
Update doc comment for AS parameter. In particular, it should be clear that the returned mask is not required to be a subset of AS. AS is effectively just a type hint.